portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -33,9 +33,51 @@ definition {
 	@ignore = "true"
 	@priority = "5"
 	test AnonymousIndividualsDoNotCountAsIndividualsForUsage {
+		ACUtils.addSiteAndPage();
 
-		// TODO LRAC-8147 Usage#AnonymousIndividualsDoNotCountAsIndividualsForUsage pending implementation
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
 
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var currentIndividuals = selenium.getText("ACUsage#USAGE_CURRENT_INDIVIDUALS");
+
+		ACUtils.launchDXP();
+
+		JSONUser.addUser(
+			userEmailAddress = "userea@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		Navigator.gotoSitePage(
+			pageName = "AC Page",
+			siteName = "Site Name");
+
+		ACUtils.closeAllSessions();
+
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var newIndividuals = selenium.getText("ACUsage#USAGE_CURRENT_INDIVIDUALS");
+
+		if ("${currentIndividuals}" == "${newIndividuals}") {
+			AssertTextEquals(
+				locator1 = "ACUsage#USAGE_CURRENT_INDIVIDUALS",
+				value1 = "${currentIndividuals}");
+		}
```

Comment:
I think adding a user is not needed since you will need to do the activity anonymously